### ISSUE TITLE
18.11.4.9: Base characteristic of get_pointer not explicit

### ISSUE THREAD

**Issue by hubert-reinterpretcast** [cplusplus/reflection-ts#83]
If the intent is to specialize integral_constant with a non-integral type, then this should be clear in the text.

**Comment by hubert-reinterpretcast** [comment]
Ditto the `Function` specialization of this template.

**Comment by Axel-Naumann** [comment]
I agree it's a wording defect; I believe it can be handled post-publication though as the intent is clear:

> with a static data member named value of type X and value x, where...
